activities.json

```diff
@@ -551,6 +551,18 @@
     "title": "Trust Token API",
     "url": "https://github.com/WICG/trust-token-api/blob/master/README.md"
   },
+  {
+    "ciuName": null,
+    "description": "An API that allows applications to lock down powerful APIs to only accept non-spoofable, typed values in place of strings to prevent vulnerabilities caused by using these APIs with attacker-controlled inputs.",
+    "id": "trusted-types",
+    "mozBugUrl": null,
+    "mozPosition": "non-harmful",
+    "mozPositionDetail": "The API does not seem suitable for the long tail of sites.",
```

Comment:
```suggestion
    "mozPositionDetail": "The API could be used to harden sites against certain cross-site scripting issues, but it is sufficiently complex that we are concerned that it will not be suitable for many sites.",
```